app/javascript/components/modals/CLIWalkthroughModal.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import { CLIWalkthrough, GraphicalIcon } from '../common'
+import { Modal, ModalProps } from './Modal'
+
+export const CLIWalkthroughModal = ({
+  html,
+  ...props
+}: Omit<ModalProps, 'className'> & { html: string }): JSX.Element => {
```

Comment:
@kntsoriano The `className` should always be specified within the modal itself (see L10) rather than passed into the Modal. So when you add new modals, can you follow this pattern pls.